girder/api/rest.py

```diff
@@ -73,6 +76,51 @@ def getApiUrl(url=None):
     return url[:idx + 7]
 
 
+def iterBody(length=READ_BUFFER_LEN, strictLength=False):
+    """
+    This is a generator that will read the request body a chunk at a time and
+    yield each chunk, abstracting details of the underlying HTTP server. This
+    function works regardless of whether the body was sent with a Content-Length
+    or using Transfer-Encoding: chunked, but the behavior is slightly different
+    in each case.
+
+    If `Content-Length` is provided, the `length` parameter is used to read the
+    body in chunks up to size `length`. This will block until end of stream or
+    the specified number of bytes is ready.
+
+    If `Transfer-Encoding: chunked` is used, the `length` parameter is ignored
+    by default the generator yields each chunk that is sent in the request,
```

Comment:
this phrasing is a bit choppy, maybe `parameter is ignored by default; the generator` or `parameter is ignored by default.  The generator` ?